### DISCUSSION TITLE
Disable Link Statistics

### DISCUSSION THREAD

**Discussion by Brulijam** [JulianPrieber/littlelink-custom#680] [Feature requests]
Hi,
I would love to use Linkstack without tracking how many people click on my links.
Currently links are like this: https://[instance]/going/[id]?[destination_url] and I would want them like this: [destination_url]

**Comment by JulianPrieber** [comment]
Already implemented in the latest version.
We now have both see: https://github.com/LinkStackOrg/LinkStack/releases/tag/v4.7.0#1
